webapp/messages/en.json

```
@@ -234,6 +234,14 @@
       "subtitle": "Stake assets to earn points and rewards.",
       "title": "Stake"
     },
+    "toast": {
+      "go-staking-dashboard": "Go to the staking dashboard",
+      "here-your-stake-tx": "Here’s your {type} transaction TX:",
```

I spoke with @rmilrad  because this text seemed incorrect  to time. TX reads as Transaction, so it seems redundant

```suggestion
      "here-your-stake-tx": "Here’s your {type} transaction:",
```

The Spanish version would be `Aquí está su transacción de {type}"`

---

Fixed. Thanks!